### PR TITLE
Bug 1602968 - Push Health better tab title and tmp favicon

### PR DESCRIPTION
**Pull request by camd**
I created a couple simple `favicon` images based on `FontAwesome` for now.  I have a couple sketches in [Bug 1602968](https://bugzilla.mozilla.org/show_bug.cgi?id=1602968) that I think might be better.  But these are OK for now.  

 I also included the Need Investigation count in the title, like we do for Unclassified Failures in TH proper.  This is updated whenever PH updates its data, so you can just watch the tab to see if it turns green.

push-health-broken

push-health-ok

### CODE REVIEW

File: ui/push-health/Health.jsx [hunk 1]
```diff
@@ -37,6 +39,7 @@ export default class Health extends React.PureComponent {
   constructor(props) {
     super(props);
 
+    document.title = 'Push Health';
```
Comment (sarah-clements) [inline ui/push-health/Health.jsx]: You could also use react helmet for that.

Comment (sarah-clements) [reply ui/push-health/Health.jsx]: Maybe even for the icon.

Comment (camd) [reply ui/push-health/Health.jsx]: Agreed.  Fixed.

**Comment by camd** [comment]
Fixed to use our default colors for red and green.

**Pull request merged**